src/main/java/org/openpnp/gui/tablemodel/FeedersTableModel.java

```diff
@@ -103,6 +116,9 @@ public Object getValueAt(int row, int col) {
             case 1:
                 return feeders.get(row).getClass().getSimpleName();
             case 2: {
+                if(feeders.get(row).getClass().getSimpleName().compareTo("ReferenceFeederSlot")==0)
```

Comment:
We can't have implementation specific code in the generic UI. We're going to have to come up with another way to do this. Maybe an API change. Will comment further after the code review.